### ISSUE TITLE
PokeBuddy on VirtualBox

### ISSUE THREAD

**Issue by iperuber** [finndev/PokeBuddy#302]
Hi everyone,

[ENVIRONMENT FAILED]
I have installed bother windows 7 32-bit and windows 7 64-bit in VirtualBox on my mac-pc. 
PokeBuddy failed to start, there is any setting to do? 

[ENVIRONMENT OK]
I have used PokeBuddy on windows-pc with windows 7 64-bit and it works without any proxy settings (some days ago)

[QUESTION]
What change for PokeBuddy to run on windows-pd instead of virtual machine?

Tnx u for support,
Marco

**Comment by sqrcbc** [comment]
it WOULD run
if the bot would work ;)

**Comment by andreimanu** [comment]
As far as I understood, the bot is written in .Net, so make sure you install the latest dot net framework.

**Comment by iperuber** [comment]
ok andreimanu, I'll try this evening. I thought the same thing. Tnx u

**Comment by andreimanu** [comment]
No probs. However, remember that the bot is currently not working, check #218 and https://www.reddit.com/r/pokemongodev/comments/4w1cvr/pokemongo_current_api_status/ for updates ^^

**Comment by iperuber** [comment]
yes yes, I know. But this problem not depends on the change of Niantec's api. It'll work on the next release of PokeBuddy.
Tnx u,
M

**Comment by iperuber** [comment]
I confirm the requirement Microsoft Framework .NET 4.5.2 or later

M
